Sources/SentryCrash/Recording/SentryBinaryImageCacheCallbacks.m

```diff
@@ -0,0 +1,30 @@
+#import "SentryBinaryImageCacheCallbacks.h"
+#import "SentryDependencyContainer.h"
+#import "SentryInternalDefines.h"
+#import "SentrySwift.h"
+
+// Used to call SentryDependencyContainer, since it isn't available to swift yet
+void
+binaryImageWasAdded(const SentryCrashBinaryImage *_Nullable image)
+{
+    if (image == NULL) {
+        SENTRY_LOG_WARN(@"The image is NULL. Can't add NULL to cache.");
+        return;
+    }
+    [SentryDependencyContainer.sharedInstance.binaryImageCache binaryImageAdded:image->name
```

Comment:
Note for future reference: We really need to find a way to get rid of SentryDependencyContainer.sharedInstance and instead configure a reference somehow else. Out of scope for this PR.